app/views/static/index.html.erb

```diff
@@ -13,8 +13,18 @@
     <%= link_to "Game #{game.id}", game_path(game) %> 
     <br>
   </h2>
- <% end %>
-
-<br>
+<% end %>
 
+<h1> Current Games </h1>
```

Comment:
Small suggestion: I was thinking that as a user, when I log in I would want to see all my current games before seeing available games. Other than that everything looks good!